android/java/org/chromium/chrome/browser/rewards/tipping/RewardsTippingBannerFragment.java

```diff
@@ -0,0 +1,341 @@
+/**
+ * Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package org.chromium.chrome.browser.rewards.tipping;
+
+import android.annotation.SuppressLint;
+import android.app.Activity;
+import android.os.Bundle;
+import android.text.TextUtils;
+import android.text.method.ScrollingMovementMethod;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.fragment.app.Fragment;
+
+import com.bumptech.glide.Glide;
```

Comment:
Can we remove glide usage here ? we need to move away from glide at some point so it's better to remove any usage we have.

Reply:
This code is old, just moved from activity to fragment.
Do you want to use ImageLoader here?
Last time I tested same thing Image Loader is very slow and caching is not working.
Filed issue : https://github.com/brave/brave-browser/issues/30136
Please let me know if anything update happened in ImageLoader recently.

Reply:
let's update it with ImageLoader and add details to the issue if we have any performance issues. so we can look into it separately.

Reply:
Sure, I will implement ImageLoader, even it's slow and caching is not working.
I will update details in the issue..

Reply:
@deeppandya 
Now it's Updated with ImageLoader
https://github.com/brave/brave-core/assets/32419898/76905bc4-404e-4445-bf15-b06824fcbc3e